packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx

```diff
@@ -264,20 +281,30 @@ const AddTilesToDashboardModal: FC<AddTilesToDashboardModalProps> = ({
                                 </Button>
                             </>
                         ) : (
-                            <FormGroup
-                                label="Name new space"
-                                labelFor="chart-space"
-                            >
-                                <InputGroup
-                                    id="chart-space"
-                                    type="text"
-                                    value={newSpaceName}
-                                    onChange={(e) =>
-                                        setNewSpaceName(e.target.value)
-                                    }
-                                    placeholder="eg. KPIs"
-                                />
-                            </FormGroup>
+                            <>
+                                <FormGroup
+                                    label="Name new space"
+                                    labelFor="chart-space"
+                                >
+                                    <InputGroup
+                                        id="chart-space"
+                                        type="text"
+                                        value={newSpaceName}
+                                        onChange={(e) =>
+                                            setNewSpaceName(e.target.value)
+                                        }
+                                        placeholder="eg. KPIs"
+                                    />
+                                </FormGroup>
+                                <Button
+                                    minimal
+                                    icon="arrow-left"
+                                    intent="primary"
+                                    onClick={() => setIsCreatingnewSpace(false)}
```

Comment:
```suggestion
                                    onClick={() => setIsCreatingNewSpace(false)}
```

would you mind making this change everywhere please? unless it's being referenced a LOT 😅

Reply:
not sure what you mean

Reply:
sure, `setIsCreatingnewSpace` is not correctly camalCased, the `n` in `new` is lowercase!

Reply:
got you! fixed ✅